geode-core/src/main/java/org/apache/geode/cache/configuration/RegionAttributesType.java

```diff
@@ -340,13 +340,13 @@
   @XmlElement(name = "value-constraint", namespace = "http://geode.apache.org/schema/cache")
   protected String valueConstraint;
   @XmlElement(name = "region-time-to-live", namespace = "http://geode.apache.org/schema/cache")
-  protected RegionAttributesType.RegionTimeToLive regionTimeToLive;
+  protected ExpirationAttributesType regionTimeToLive;
```

Comment:
👍 for this change. Assuming it doesn't affect the schema?

Reply:
No. I had a test in RegionConfigTest to make sure the xml stays the same.